components/namespace-lister/production/base/patches/with_header_auth_email.yaml

```
@@ -0,0 +1,5 @@
+- op: add
+  path: /spec/template/spec/containers/0/env/-
+  value:
+    name: AUTH_USERNAME_HEADER
+    value: X-Email
```

production is already explicitly using usernames, let's use `X-User` here
* https://github.com/redhat-appstudio/infra-deployments/blob/265a37d5659ad07ae0951ecf7033cad2a4ad9e00/components/konflux-ui/production/base/proxy/proxy.yaml#L71
* https://github.com/redhat-appstudio/infra-deployments/blob/265a37d5659ad07ae0951ecf7033cad2a4ad9e00/components/konflux-ui/production/base/proxy/nginx.conf#L69

```suggestion
    value: X-User
```

We also need to update the filename to `with_header_auth_user.yaml`

---

Fixed in 192f5512.